app/decorators/user_decorator.rb

```diff
@@ -1,3 +1,15 @@
 class UserDecorator < BaseDecorator
   delegate :id, :username, :email
+
+  def exist?
+    object.id.present?
+  end
+
+  def articles_length
+    object.articles.length
```

Comment:
it's better to use .count here because 'length' will return size of array in RAM, but .count will create SQL query to count articles